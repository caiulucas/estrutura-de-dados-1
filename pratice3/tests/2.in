6
Breno 163
Arthur 91
Olivia 88
AnaLaura 134
Emanuel 92
Betina 19
