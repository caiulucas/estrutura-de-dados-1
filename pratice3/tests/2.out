Time 1
[L] Breno 163
[P] Emanuel 92
[E] Olivia 88

Time 2
[L] AnaLaura 134
[P] Arthur 91
[E] Betina 19

